python/prophet/plot.py

```diff
@@ -89,7 +89,13 @@ def plot(
     ax.set_ylabel(ylabel)
     if include_legend:
         ax.legend()
-    fig.tight_layout()
+    # Check if user passed in an ax
+    if ax is None:
```

Comment:
We first check if `ax` is passed in lines 66-70, and assign one if it's not. So we'd have to create a variable there to track whether we're dealing with a user-provided ax.